### ISSUE TITLE
Buffer overflow when loading Spack on Cray with .bash_rc

### ISSUE THREAD

**Issue by adamryczkowski** [spack/spack#6862]
### Expected Result

Spack loaded into the bash session automatically.

### Actual Result

```
#.bash_rc line: . tmp/spack/share/spack/setup-env.sh 
*** buffer overflow detected ***: /opt/cray/pe/modules/3.2.10.6/bin/modulecmd terminated
======= Backtrace: =========
/lib64/libc.so.6(+0x721af)[0x2adc1373e1af]
/lib64/libc.so.6(__fortify_fail+0x37)[0x2adc137c5dc7]
/lib64/libc.so.6(+0xf8050)[0x2adc137c4050]
/lib64/libc.so.6(+0xf9d37)[0x2adc137c5d37]
/usr/lib64/libtcl8.6.so(+0x136a2b)[0x2adc12f50a2b]
/lib64/libpthread.so.0(+0x8744)[0x2adc13a77744]
/lib64/libc.so.6(clone+0x6d)[0x2adc137b5aad]
======= Memory map: ========
00400000-00422000 r-xp 00000000 08:11 2159803747                         /opt/cray/pe/modules/3.2.10.6/bin/modulecmd
00621000-00622000 r--p 00021000 08:11 2159803747                         /opt/cray/pe/modules/3.2.10.6/bin/modulecmd
00622000-00625000 rw-p 00022000 08:11 2159803747                         /opt/cray/pe/modules/3.2.10.6/bin/modulecmd
00625000-0062f000 rw-p 00000000 00:00 0 
00ced000-07dc4000 rw-p 00000000 00:00 0                                  [heap]
2adc12bf7000-2adc12c18000 r-xp 00000000 08:02 523380                     /lib64/ld-2.22.so
2adc12c18000-2adc12c1b000 rw-p 00000000 00:00 0 
2adc12c1b000-2adc12c1c000 rw-p 00000000 00:00 0 
2adc12c6c000-2adc12d70000 rw-p 00000000 00:00 0 
2adc12e17000-2adc12e18000 r--p 00020000 08:02 523380                     /lib64/ld-2.22.so
2adc12e18000-2adc12e19000 rw-p 00021000 08:02 523380                     /lib64/ld-2.22.so
2adc12e19000-2adc12e1a000 rw-p 00000000 00:00 0 
2adc12e1a000-2adc12fbb000 r-xp 00000000 08:02 1329350                    /usr/lib64/libtcl8.6.so
2adc12fbb000-2adc131ba000 ---p 001a1000 08:02 1329350                    /usr/lib64/libtcl8.6.so
2adc131ba000-2adc131c8000 r--p 001a0000 08:02 1329350                    /usr/lib64/libtcl8.6.so
2adc131c8000-2adc131ca000 rw-p 001ae000 08:02 1329350                    /usr/lib64/libtcl8.6.so
2adc131ca000-2adc131cb000 rw-p 00000000 00:00 0 
2adc131cb000-2adc131cd000 r-xp 00000000 08:02 523402                     /lib64/libdl-2.22.so
2adc131cd000-2adc133cd000 ---p 00002000 08:02 523402                     /lib64/libdl-2.22.so
2adc133cd000-2adc133ce000 r--p 00002000 08:02 523402                     /lib64/libdl-2.22.so
2adc133ce000-2adc133cf000 rw-p 00003000 08:02 523402                     /lib64/libdl-2.22.so
2adc133cf000-2adc134cb000 r-xp 00000000 08:02 523421                     /lib64/libm-2.22.so
2adc134cb000-2adc136ca000 ---p 000fc000 08:02 523421                     /lib64/libm-2.22.so
2adc136ca000-2adc136cb000 r--p 000fb000 08:02 523421                     /lib64/libm-2.22.so
2adc136cb000-2adc136cc000 rw-p 000fc000 08:02 523421                     /lib64/libm-2.22.so
2adc136cc000-2adc13865000 r-xp 00000000 08:02 523388                     /lib64/libc-2.22.so
2adc13865000-2adc13a65000 ---p 00199000 08:02 523388                     /lib64/libc-2.22.so
2adc13a65000-2adc13a69000 r--p 00199000 08:02 523388                     /lib64/libc-2.22.so
2adc13a69000-2adc13a6b000 rw-p 0019d000 08:02 523388                     /lib64/libc-2.22.so
2adc13a6b000-2adc13a6f000 rw-p 00000000 00:00 0 
2adc13a6f000-2adc13a87000 r-xp 00000000 08:02 523445                     /lib64/libpthread-2.22.so
2adc13a87000-2adc13c86000 ---p 00018000 08:02 523445                     /lib64/libpthread-2.22.so
2adc13c86000-2adc13c87000 r--p 00017000 08:02 523445                     /lib64/libpthread-2.22.so
2adc13c87000-2adc13c88000 rw-p 00018000 08:02 523445                     /lib64/libpthread-2.22.so
2adc13c88000-2adc13c8c000 rw-p 00000000 00:00 0 
2adc13c8c000-2adc13c8d000 ---p 00000000 00:00 0 
2adc13c8d000-2adc13e8d000 rw-p 00000000 00:00 0 
2adc13e8d000-2adc13e94000 r-xp 00000000 08:02 523431                     /lib64/libnss_compat-2.22.so
2adc13e94000-2adc14093000 ---p 00007000 08:02 523431                     /lib64/libnss_compat-2.22.so
2adc14093000-2adc14094000 r--p 00006000 08:02 523431                     /lib64/libnss_compat-2.22.so
2adc14094000-2adc14095000 rw-p 00007000 08:02 523431                     /lib64/libnss_compat-2.22.so
2adc14095000-2adc140aa000 r-xp 00000000 08:02 523430                     /lib64/libnsl-2.22.so
2adc140aa000-2adc142a9000 ---p 00015000 08:02 523430                     /lib64/libnsl-2.22.so
2adc142a9000-2adc142aa000 r--p 00014000 08:02 523430                     /lib64/libnsl-2.22.so
2adc142aa000-2adc142ab000 rw-p 00015000 08:02 523430                     /lib64/libnsl-2.22.so
2adc142ab000-2adc142ad000 rw-p 00000000 00:00 0 
2adc142ad000-2adc142b7000 r-xp 00000000 08:02 523437                     /lib64/libnss_nis-2.22.so
2adc142b7000-2adc144b6000 ---p 0000a000 08:02 523437                     /lib64/libnss_nis-2.22.so
2adc144b6000-2adc144b7000 r--p 00009000 08:02 523437                     /lib64/libnss_nis-2.22.so
2adc144b7000-2adc144b8000 rw-p 0000a000 08:02 523437                     /lib64/libnss_nis-2.22.so
2adc144b8000-2adc144c3000 r-xp 00000000 08:02 523434                     /lib64/libnss_files-2.22.so
2adc144c3000-2adc146c2000 ---p 0000b000 08:02 523434                     /lib64/libnss_files-2.22.so
2adc146c2000-2adc146c3000 r--p 0000a000 08:02 523434                     /lib64/libnss_files-2.22.so
2adc146c3000-2adc146c4000 rw-p 0000b000 08:02 523434                     /lib64/libnss_files-2.22.so
2adc146c4000-2adc146ca000 rw-p 00000000 00:00 0 
2adc146ca000-2adc146d2000 r-xp 00000000 08:02 523439                     /lib64/libnss_sss.so.2
2adc146d2000-2adc148d1000 ---p 00008000 08:02 523439                     /lib64/libnss_sss.so.2
2adc148d1000-2adc148d2000 r--p 00007000 08:02 523439                     /lib64/libnss_sss.so.2
2adc148d2000-2adc148d3000 rw-p 00008000 08:02 523439                     /lib64/libnss_sss.so.2
2adc148d3000-2adc14ad5000 rw-p 00000000 00:00 0 
2adc14bd6000-2adc166f1000 rw-p 00000000 00:00 0 
2adc166f1000-2adc16707000 r-xp 00000000 08:02 523411                     /lib64/libgcc_s.so.1
2adc16707000-2adc16906000 ---p 00016000 08:02 523411                     /lib64/libgcc_s.so.1
2adc16906000-2adc16907000 r--p 00015000 08:02 523411                     /lib64/libgcc_s.so.1
2adc16907000-2adc16908000 rw-p 00016000 08:02 523411                     /lib64/libgcc_s.so.1
2adc18000000-2adc18021000 rw-p 00000000 00:00 0 
2adc18021000-2adc1c000000 ---p 00000000 00:00 0 
7ffd4d58c000-7ffd4d5ae000 rw-p 00000000 00:00 0                          [stack]
7ffd4d5fa000-7ffd4d5fd000 r--p 00000000 00:00 0                          [vvar]
7ffd4d5fd000-7ffd4d5ff000 r-xp 00000000 00:00 0                          [vdso]
ffffffffff600000-ffffffffff601000 r-xp 00000000 00:00 0                  [vsyscall]
*** buffer overflow detected ***: /opt/cray/pe/modules/3.2.10.6/bin/modulecmd terminated
*** buffer overflow detected ***: /opt/cray/pe/modules/3.2.10.6/bin/modulecmd terminated
======= Backtrace: =========
/lib64/libc.so.6(+0x721af)[0x2b0b1b4851af]
/lib64/libc.so.6(__fortify_fail+0x37)[0x2b0b1b50cdc7]
/lib64/libc.so.6(+0xf8050)[0x2b0b1b50b050]
/lib64/libc.so.6(+0xf9d37)[0x2b0b1b50cd37]
/usr/lib64/libtcl8.6.so(+0x136a2b)[0x2b0b1ac97a2b]
/lib64/libpthread.so.0(+0x8744)[0x2b0b1b7be744]
/lib64/libc.so.6(clone+0x6d)[0x2b0b1b4fcaad]
======= Memory map: ========
00400000-00422000 r-xp 00000000 08:11 2159803747                         /opt/cray/pe/modules/3.2.10.6/bin/modulecmd
00621000-00622000 r--p 00021000 08:11 2159803747                         /opt/cray/pe/modules/3.2.10.6/bin/modulecmd
00622000-00625000 rw-p 00022000 08:11 2159803747                         /opt/cray/pe/modules/3.2.10.6/bin/modulecmd
00625000-0062f000 rw-p 00000000 00:00 0 
014cc000-0171f000 rw-p 00000000 00:00 0                                  [heap]
2b0b1a93e000-2b0b1a95f000 r-xp 00000000 08:02 523380                     /lib64/ld-2.22.so
2b0b1a95f000-2b0b1a961000 rw-p 00000000 00:00 0 
2b0b1a961000-2b0b1a962000 rw-p 00000000 00:00 0 
2b0b1a9b3000-2b0b1aab8000 rw-p 00000000 00:00 0 
2b0b1ab5e000-2b0b1ab5f000 r--p 00020000 08:02 523380                     /lib64/ld-2.22.so
2b0b1ab5f000-2b0b1ab60000 rw-p 00021000 08:02 523380                     /lib64/ld-2.22.so
2b0b1ab60000-2b0b1ab61000 rw-p 00000000 00:00 0 
2b0b1ab61000-2b0b1ad02000 r-xp 00000000 08:02 1329350                    /usr/lib64/libtcl8.6.so
2b0b1ad02000-2b0b1af01000 ---p 001a1000 08:02 1329350                    /usr/lib64/libtcl8.6.so
2b0b1af01000-2b0b1af0f000 r--p 001a0000 08:02 1329350                    /usr/lib64/libtcl8.6.so
2b0b1af0f000-2b0b1af11000 rw-p 001ae000 08:02 1329350                    /usr/lib64/libtcl8.6.so
2b0b1af11000-2b0b1af12000 rw-p 00000000 00:00 0 
2b0b1af12000-2b0b1af14000 r-xp 00000000 08:02 523402                     /lib64/libdl-2.22.so
2b0b1af14000-2b0b1b114000 ---p 00002000 08:02 523402                     /lib64/libdl-2.22.so
2b0b1b114000-2b0b1b115000 r--p 00002000 08:02 523402                     /lib64/libdl-2.22.so
2b0b1b115000-2b0b1b116000 rw-p 00003000 08:02 523402                     /lib64/libdl-2.22.so
2b0b1b116000-2b0b1b212000 r-xp 00000000 08:02 523421                     /lib64/libm-2.22.so
2b0b1b212000-2b0b1b411000 ---p 000fc000 08:02 523421                     /lib64/libm-2.22.so
2b0b1b411000-2b0b1b412000 r--p 000fb000 08:02 523421                     /lib64/libm-2.22.so
2b0b1b412000-2b0b1b413000 rw-p 000fc000 08:02 523421                     /lib64/libm-2.22.so
2b0b1b413000-2b0b1b5ac000 r-xp 00000000 08:02 523388                     /lib64/libc-2.22.so
2b0b1b5ac000-2b0b1b7ac000 ---p 00199000 08:02 523388                     /lib64/libc-2.22.so
2b0b1b7ac000-2b0b1b7b0000 r--p 00199000 08:02 523388                     /lib64/libc-2.22.so
2b0b1b7b0000-2b0b1b7b2000 rw-p 0019d000 08:02 523388                     /lib64/libc-2.22.so
2b0b1b7b2000-2b0b1b7b6000 rw-p 00000000 00:00 0 
2b0b1b7b6000-2b0b1b7ce000 r-xp 00000000 08:02 523445                     /lib64/libpthread-2.22.so
2b0b1b7ce000-2b0b1b9cd000 ---p 00018000 08:02 523445                     /lib64/libpthread-2.22.so
2b0b1b9cd000-2b0b1b9ce000 r--p 00017000 08:02 523445                     /lib64/libpthread-2.22.so
2b0b1b9ce000-2b0b1b9cf000 rw-p 00018000 08:02 523445                     /lib64/libpthread-2.22.so
2b0b1b9cf000-2b0b1b9d3000 rw-p 00000000 00:00 0 
2b0b1b9d3000-2b0b1b9d4000 ---p 00000000 00:00 0 
2b0b1b9d4000-2b0b1bbd4000 rw-p 00000000 00:00 0 
2b0b1bbd4000-2b0b1bbdb000 r-xp 00000000 08:02 523431                     /lib64/libnss_compat-2.22.so
2b0b1bbdb000-2b0b1bdda000 ---p 00007000 08:02 523431                     /lib64/libnss_compat-2.22.so
2b0b1bdda000-2b0b1bddb000 r--p 00006000 08:02 523431                     /lib64/libnss_compat-2.22.so
2b0b1bddb000-2b0b1bddc000 rw-p 00007000 08:02 523431                     /lib64/libnss_compat-2.22.so
2b0b1bddc000-2b0b1bdf1000 r-xp 00000000 08:02 523430                     /lib64/libnsl-2.22.so
2b0b1bdf1000-2b0b1bff0000 ---p 00015000 08:02 523430                     /lib64/libnsl-2.22.so
2b0b1bff0000-2b0b1bff1000 r--p 00014000 08:02 523430                     /lib64/libnsl-2.22.so
2b0b1bff1000-2b0b1bff2000 rw-p 00015000 08:02 523430                     /lib64/libnsl-2.22.so
2b0b1bff2000-2b0b1bff4000 rw-p 00000000 00:00 0 
2b0b1bff4000-2b0b1bffe000 r-xp 00000000 08:02 523437                     /lib64/libnss_nis-2.22.so
2b0b1bffe000-2b0b1c1fd000 ---p 0000a000 08:02 523437                     /lib64/libnss_nis-2.22.so
2b0b1c1fd000-2b0b1c1fe000 r--p 00009000 08:02 523437                     /lib64/libnss_nis-2.22.so
2b0b1c1fe000-2b0b1c1ff000 rw-p 0000a000 08:02 523437                     /lib64/libnss_nis-2.22.so
2b0b1c1ff000-2b0b1c20a000 r-xp 00000000 08:02 523434                     /lib64/libnss_files-2.22.so
2b0b1c20a000-2b0b1c409000 ---p 0000b000 08:02 523434                     /lib64/libnss_files-2.22.so
2b0b1c409000-2b0b1c40a000 r--p 0000a000 08:02 523434                     /lib64/libnss_files-2.22.so
2b0b1c40a000-2b0b1c40b000 rw-p 0000b000 08:02 523434                     /lib64/libnss_files-2.22.so
2b0b1c40b000-2b0b1c411000 rw-p 00000000 00:00 0 
2b0b1c411000-2b0b1c419000 r-xp 00000000 08:02 523439                     /lib64/libnss_sss.so.2
2b0b1c419000-2b0b1c618000 ---p 00008000 08:02 523439                     /lib64/libnss_sss.so.2
2b0b1c618000-2b0b1c619000 r--p 00007000 08:02 523439                     /lib64/libnss_sss.so.2
2b0b1c619000-2b0b1c61a000 rw-p 00008000 08:02 523439                     /lib64/libnss_sss.so.2
2b0b1c61a000-2b0b1cc20000 rw-p 00000000 00:00 0 
2b0b1cc20000-2b0b1cc36000 r-xp 00000000 08:02 523411                     /lib64/libgcc_s.so.1
2b0b1cc36000-2b0b1ce35000 ---p 00016000 08:02 523411                     /lib64/libgcc_s.so.1
2b0b1ce35000-2b0b1ce36000 r--p 00015000 08:02 523411                     /lib64/libgcc_s.so.1
2b0b1ce36000-2b0b1ce37000 rw-p 00016000 08:02 523411                     /lib64/libgcc_s.so.1
2b0b20000000-2b0b20021000 rw-p 00000000 00:00 0 
2b0b20021000-2b0b24000000 ---p 00000000 00:00 0 
7fffed9e4000-7fffeda0b000 rw-p 00000000 00:00 0                          [stack]
7fffedb74000-7fffedb77000 r--p 00000000 00:00 0                          [vvar]
7fffedb77000-7fffedb79000 r-xp 00000000 00:00 0                          [vdso]
ffffffffff600000-ffffffffff601000 r-xp 00000000 00:00 0                  [vsyscall]
*** buffer overflow detected ***: /opt/cray/pe/modules/3.2.10.6/bin/modulecmd terminated
======= Backtrace: =========
/lib64/libc.so.6(+0x721af)[0x2b4c2e1f01af]
/lib64/libc.so.6(__fortify_fail+0x37)[0x2b4c2e277dc7]
/lib64/libc.so.6(+0xf8050)[0x2b4c2e276050]
/lib64/libc.so.6(+0xf9d37)[0x2b4c2e277d37]
/usr/lib64/libtcl8.6.so(+0x136a2b)[0x2b4c2da02a2b]
/lib64/libpthread.so.0(+0x8744)[0x2b4c2e529744]
/lib64/libc.so.6(clone+0x6d)[0x2b4c2e267aad]
======= Memory map: ========
00400000-00422000 r-xp 00000000 08:11 2159803747                         /opt/cray/pe/modules/3.2.10.6/bin/modulecmd
00621000-00622000 r--p 00021000 08:11 2159803747                         /opt/cray/pe/modules/3.2.10.6/bin/modulecmd
00622000-00625000 rw-p 00022000 08:11 2159803747                         /opt/cray/pe/modules/3.2.10.6/bin/modulecmd
00625000-0062f000 rw-p 00000000 00:00 0 
016ff000-01830000 rw-p 00000000 00:00 0                                  [heap]
2b4c2d6a9000-2b4c2d6ca000 r-xp 00000000 08:02 523380                     /lib64/ld-2.22.so
2b4c2d6ca000-2b4c2d6cc000 rw-p 00000000 00:00 0 
2b4c2d6cc000-2b4c2d6cd000 rw-p 00000000 00:00 0 
2b4c2d71e000-2b4c2d822000 rw-p 00000000 00:00 0 
2b4c2d8c9000-2b4c2d8ca000 r--p 00020000 08:02 523380                     /lib64/ld-2.22.so
2b4c2d8ca000-2b4c2d8cb000 rw-p 00021000 08:02 523380                     /lib64/ld-2.22.so
2b4c2d8cb000-2b4c2d8cc000 rw-p 00000000 00:00 0 
2b4c2d8cc000-2b4c2da6d000 r-xp 00000000 08:02 1329350                    /usr/lib64/libtcl8.6.so
2b4c2da6d000-2b4c2dc6c000 ---p 001a1000 08:02 1329350                    /usr/lib64/libtcl8.6.so
2b4c2dc6c000-2b4c2dc7a000 r--p 001a0000 08:02 1329350                    /usr/lib64/libtcl8.6.so
2b4c2dc7a000-2b4c2dc7c000 rw-p 001ae000 08:02 1329350                    /usr/lib64/libtcl8.6.so
2b4c2dc7c000-2b4c2dc7d000 rw-p 00000000 00:00 0 
2b4c2dc7d000-2b4c2dc7f000 r-xp 00000000 08:02 523402                     /lib64/libdl-2.22.so
2b4c2dc7f000-2b4c2de7f000 ---p 00002000 08:02 523402                     /lib64/libdl-2.22.so
2b4c2de7f000-2b4c2de80000 r--p 00002000 08:02 523402                     /lib64/libdl-2.22.so
2b4c2de80000-2b4c2de81000 rw-p 00003000 08:02 523402                     /lib64/libdl-2.22.so
2b4c2de81000-2b4c2df7d000 r-xp 00000000 08:02 523421                     /lib64/libm-2.22.so
2b4c2df7d000-2b4c2e17c000 ---p 000fc000 08:02 523421                     /lib64/libm-2.22.so
2b4c2e17c000-2b4c2e17d000 r--p 000fb000 08:02 523421                     /lib64/libm-2.22.so
2b4c2e17d000-2b4c2e17e000 rw-p 000fc000 08:02 523421                     /lib64/libm-2.22.so
2b4c2e17e000-2b4c2e317000 r-xp 00000000 08:02 523388                     /lib64/libc-2.22.so
2b4c2e317000-2b4c2e517000 ---p 00199000 08:02 523388                     /lib64/libc-2.22.so
2b4c2e517000-2b4c2e51b000 r--p 00199000 08:02 523388                     /lib64/libc-2.22.so
2b4c2e51b000-2b4c2e51d000 rw-p 0019d000 08:02 523388                     /lib64/libc-2.22.so
2b4c2e51d000-2b4c2e521000 rw-p 00000000 00:00 0 
2b4c2e521000-2b4c2e539000 r-xp 00000000 08:02 523445                     /lib64/libpthread-2.22.so
2b4c2e539000-2b4c2e738000 ---p 00018000 08:02 523445                     /lib64/libpthread-2.22.so
2b4c2e738000-2b4c2e739000 r--p 00017000 08:02 523445                     /lib64/libpthread-2.22.so
2b4c2e739000-2b4c2e73a000 rw-p 00018000 08:02 523445                     /lib64/libpthread-2.22.so
2b4c2e73a000-2b4c2e73e000 rw-p 00000000 00:00 0 
2b4c2e73e000-2b4c2e73f000 ---p 00000000 00:00 0 
2b4c2e73f000-2b4c2e93f000 rw-p 00000000 00:00 0 
2b4c2e93f000-2b4c2e946000 r-xp 00000000 08:02 523431                     /lib64/libnss_compat-2.22.so
2b4c2e946000-2b4c2eb45000 ---p 00007000 08:02 523431                     /lib64/libnss_compat-2.22.so
2b4c2eb45000-2b4c2eb46000 r--p 00006000 08:02 523431                     /lib64/libnss_compat-2.22.so
2b4c2eb46000-2b4c2eb47000 rw-p 00007000 08:02 523431                     /lib64/libnss_compat-2.22.so
2b4c2eb47000-2b4c2eb5c000 r-xp 00000000 08:02 523430                     /lib64/libnsl-2.22.so
2b4c2eb5c000-2b4c2ed5b000 ---p 00015000 08:02 523430                     /lib64/libnsl-2.22.so
2b4c2ed5b000-2b4c2ed5c000 r--p 00014000 08:02 523430                     /lib64/libnsl-2.22.so
2b4c2ed5c000-2b4c2ed5d000 rw-p 00015000 08:02 523430                     /lib64/libnsl-2.22.so
2b4c2ed5d000-2b4c2ed5f000 rw-p 00000000 00:00 0 
2b4c2ed5f000-2b4c2ed69000 r-xp 00000000 08:02 523437                     /lib64/libnss_nis-2.22.so
2b4c2ed69000-2b4c2ef68000 ---p 0000a000 08:02 523437                     /lib64/libnss_nis-2.22.so
2b4c2ef68000-2b4c2ef69000 r--p 00009000 08:02 523437                     /lib64/libnss_nis-2.22.so
2b4c2ef69000-2b4c2ef6a000 rw-p 0000a000 08:02 523437                     /lib64/libnss_nis-2.22.so
2b4c2ef6a000-2b4c2ef75000 r-xp 00000000 08:02 523434                     /lib64/libnss_files-2.22.so
2b4c2ef75000-2b4c2f174000 ---p 0000b000 08:02 523434                     /lib64/libnss_files-2.22.so
2b4c2f174000-2b4c2f175000 r--p 0000a000 08:02 523434                     /lib64/libnss_files-2.22.so
2b4c2f175000-2b4c2f176000 rw-p 0000b000 08:02 523434                     /lib64/libnss_files-2.22.so
2b4c2f176000-2b4c2f17c000 rw-p 00000000 00:00 0 
2b4c2f17c000-2b4c2f184000 r-xp 00000000 08:02 523439                     /lib64/libnss_sss.so.2
2b4c2f184000-2b4c2f383000 ---p 00008000 08:02 523439                     /lib64/libnss_sss.so.2
2b4c2f383000-2b4c2f384000 r--p 00007000 08:02 523439                     /lib64/libnss_sss.so.2
2b4c2f384000-2b4c2f385000 rw-p 00008000 08:02 523439                     /lib64/libnss_sss.so.2
2b4c2f385000-2b4c2f587000 rw-p 00000000 00:00 0 
2b4c2f587000-2b4c2f59d000 r-xp 00000000 08:02 523411                     /lib64/libgcc_s.so.1
2b4c2f59d000-2b4c2f79c000 ---p 00016000 08:02 523411                     /lib64/libgcc_s.so.1
2b4c2f79c000-2b4c2f79d000 r--p 00015000 08:02 523411                     /lib64/libgcc_s.so.1
2b4c2f79d000-2b4c2f79e000 rw-p 00016000 08:02 523411                     /lib64/libgcc_s.so.1
2b4c30000000-2b4c30021000 rw-p 00000000 00:00 0 
2b4c30021000-2b4c34000000 ---p 00000000 00:00 0 
7ffd4d7ed000-7ffd4d810000 rw-p 00000000 00:00 0                          [stack]
7ffd4d9ba000-7ffd4d9bd000 r--p 00000000 00:00 0                          [vvar]
7ffd4d9bd000-7ffd4d9bf000 r-xp 00000000 00:00 0                          [vdso]
ffffffffff600000-ffffffffff601000 r-xp 00000000 00:00 0                  [vsyscall]
```

It works fine, if I simply load the spack interactively after login.

Deleting the `~/.spack` folder didn't help.

**Comment by adamjstewart** [comment]
Duplicate of #5114. @skosukhin @pramodskumbhar did either of you ever figure out a way around this?

**Comment by mamelara** [comment]
Okay since no one has responded I'll write up a fix for this as suggested in #5114

**Comment by adamjstewart** [comment]
#5114 has been closed, so I'm assuming this duplicate issue can also be closed.
